lib/sandthorn_sequel_projection.rb

```diff
@@ -22,7 +21,7 @@ class << self
     require 'delegate'
     extend Forwardable
 
-    def_delegators :configuration, :batch_size, :event_stores
+    def_delegators :configuration, :batch_size, :event_store
```

Comment:
There seems to already be a code seam were you can implement this: the EventStore class

> 6 maj 2016 kl. 15:21 skrev Morgan Hallgren <notifications@github.com>:
> 
> In lib/sandthorn_sequel_projection.rb:
> 
> > @@ -22,7 +21,7 @@ class << self
> >      require 'delegate'
> >      extend Forwardable
> >  
> > -    def_delegators :configuration, :batch_size, :event_stores
> > +    def_delegators :configuration, :batch_size, :event_store
> The Sandthorn coupling was needed before the serialization was moved to the drivers.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>